http/rest-client/src/main/java/io/quarkus/ts/http/restclient/RestInterface.java

```diff
@@ -24,4 +24,9 @@ public interface RestInterface {
     @Path("/json")
     @Produces(MediaType.APPLICATION_JSON)
     Book getAsJson();
+
+    @GET
+    @Path("/notFound")
```

Comment:
I don't see a point of exposing this endpoint when you never actually call it, please mention you are calling `given().get("/notFound")` not `given().get("/client/book/notFound")`.

Reply:
You are right!. Even I don't need an endpoint over `ClientBookResource`. The issue could be reproduced with a simple notFound.